katalog/tigera/eks-policy-only/kustomization.yaml

```diff
@@ -8,3 +8,17 @@ kind: Kustomization
 resources:
   - ../operator
   - calico-crs.yaml
+
+patches:
```

Comment:
We don't need a patch, just change the file calico-crs.yaml

Reply:
this ☝️